### ISSUE TITLE
<Button type="submit" waitForInkTransition /> submits form twice on click

### ISSUE THREAD

**Issue by mctep** [mlaursen/react-md#307]
Hello! Thank you for your job. 👍 

### Description

I try to use `ExpansionPanel` as `form` with submitting by pressing `Enter` like this:

```jsx
<ExpansionPanel
  component="form"
  onSubmit={ () => console.log('handleSubmit'); }
  saveType="submit"
/>
```

`InkContainer` component (from `Button < DialogFooter < PanelContent < ExpansionPanel`) emits `Button` click by `waitForInkTransition` prop. So it causes the second submit event on click. I think it is an unexpected behaviour :)

It is not a big problem but at the moment I have to:
* remove `saveType="submit"`;
* add hidden submit button;
* add yet `onSave` handler for submitting.

**Comment by mlaursen** [comment]
Thanks! My next release should be removing that `waitForInkTransition` thing. It was something I thought flowed nicer, but that delay is killer from a UX perspective. It should hopefully be resolved by then.

**Comment by mlaursen** [comment]
Cool. Looks like this was fixed in #210 after testing this case again.
